tests/h/search/query_test.py

```diff
@@ -24,7 +24,7 @@ class TestBuilder(object):
         # Default sort field should be "updated".
         (None, "asc", [2, 0, 1]),
     ])
-    def test_it_sorts_annotations(self, Annotation, search, sort_key, order, expected_order):
+    def test_it_sorts_annotations(self, Annotation, search, sort_key, order, expected_order, pyramid_request):
```

Comment:
I think would be preferable to use `pytest.mark.usefixtures` here to use the `pyramid_request` fixture rather than adding it as a parameter, since that makes it clearer that the fixture is being used for its side effects.

_However_ you don't actually need to explicitly this fixture in any tests that already use `search` because the `search` fixture is already parametrized by `pyramid_request`.